Technical-Steering-Committee/logging.md

```diff
@@ -0,0 +1,56 @@
+# Message Management
+
+## How do we log
+
+### Message ID Components (message prefixes)
+
+ZWE - Zowe, This should be used as default for all Zowe messages. 
+
+### Zowe Projects
+
+There are server and client side components that are logging separately as such we may have duplication of logging for Server side and Client side. 
+
+#### Server Side
+
+A - API Mediation Layer
+D - Zowe App Framework
+S - Zowe System Services
+
+#### Client Side 
+
+C - Zowe CLI
+E - Zowe VSCode Explorer
+H - Zowe Chat
+I - Zowe Intellij Explorer
+J - Client Java SDK
+K - Client Kotlin SDK
+N - Client Node.js SDK
+P - Client Python SDK
+
+### Message Formats
+
+ZWEcnnnnt, ZWEcsnnnt or ZWEcsxnnt {Actual message} where:
```

Comment:
perhaps a note that WTO messages should be 8 characters with proper route and descriptor codes according to the message.  we probably don't want to duplicate what's proper and rather refer to IBM publications?